i18n/i18n.go

```diff
@@ -56,14 +57,18 @@ func (i18n *I18n) AddTransaltion(translation *Translation) {
 	i18n.Translations[translation.Locale][translation.Key] = translation
 }
 
-func (i18n *I18n) SaveTransaltion(translation *Translation) {
+func (i18n *I18n) SaveTranslation(translation *Translation) (err error) {
```

Comment:
I was just about to refactor `SaveTranslation` to call `AddTranslation` on #30 :crying_cat_face: